.github/actions/javascript/awaitStagingDeploys/awaitStagingDeploys.ts

```diff
@@ -39,28 +42,28 @@ function run() {
         ])
             .then((responses) => {
                 const workflowRuns = responses[0].data.workflow_runs;
-                if (!tag) {
+                if (!tag && typeof responses[1] === 'object') {
                     workflowRuns.push(...responses[1].data.workflow_runs);
                 }
                 return workflowRuns;
             })
-            .then((workflowRuns) => (currentStagingDeploys = _.filter(workflowRuns, (workflowRun) => workflowRun.status !== 'completed')))
+            .then((workflowRuns) => (currentStagingDeploys = workflowRuns.filter((workflowRun) => workflowRun.status !== 'completed')))
             .then(() =>
                 console.log(
-                    _.isEmpty(currentStagingDeploys)
+                    !currentStagingDeploys.length
                         ? 'No current staging deploys found'
                         : `Found ${currentStagingDeploys.length} staging deploy${currentStagingDeploys.length > 1 ? 's' : ''} still running...`,
                 ),
             );
     console.info('[awaitStagingDeploys] run() throttleFunc', throttleFunc);
 
     return promiseDoWhile(
-        () => !_.isEmpty(currentStagingDeploys),
-        _.throttle(
+        () => currentStagingDeploys.length,
```

Comment:
```suggestion
        () => !!currentStagingDeploys.length,
```

Shouldn't be like this?